redis-utils-lib/project.clj

```diff
@@ -42,16 +44,15 @@
              ;; The following profile is overriden on the build server or in the user's
              ;; ~/.lein/profiles.clj file.
              :internal-repos {}
-             :kaocha {:dependencies [[lambdaisland/kaocha "1.0.700"]
-                                     [lambdaisland/kaocha-cloverage "1.0.63"]
+             :kaocha {:dependencies [[lambdaisland/kaocha "1.0.732"]
+                                     [lambdaisland/kaocha-cloverage "1.0.75"]
                                      [lambdaisland/kaocha-junit-xml "0.0.76"]]}}
   :aliases {"bikeshed" ["with-profile" "lint" "bikeshed" "--max-line-length=100"]
             "check-deps" ["with-profile" "lint" "ancient" ":all"]
             "check-sec" ["with-profile" "security" "dependency-check"]
             "eastwood" ["with-profile" "lint" "eastwood" "{:namespaces [:source-paths]}"]
-            "pull-docker-images" ["shell" "docker" "pull" "redis:6"]
+            "pull-docker-images" ["shell" "docker" "pull" ~(str "redis:" redis-version)]
```

Comment:
This feels a bit overkill, and I think the only reason I didn't feel that way with the elasticsearch version is there wasn't the need for a `str` call. Are the redis and elasticsearch versions being called elsewhere? If not I don't really see the benefit of this, even if it is nifty